models/post.js

```diff
@@ -1,10 +1,18 @@
-'use strict';
+var slug = require('slug');
+
 module.exports = function(sequelize, DataTypes) {
   var Post = sequelize.define('Post', {
     title: DataTypes.STRING,
     slug: DataTypes.STRING,
     content: DataTypes.TEXT
   }, {
+    hooks: {
```

Comment:
because they are functions "hooked" to model lifecycles events, they run in the event they are described in: http://docs.sequelizejs.com/en/v3/docs/hooks/